Classes/Notifications/NotificationsViewController.swift

```diff
@@ -304,4 +307,13 @@ BaseListViewController2EmptyDataSource {
     func didDoubleTapTab() {
         didSingleTapTab()
     }
+    
+    // MARK: ReviewGithubAccessDelegate
+    func reviewGithubAccessButtonTapped() {
+        //copied/pasted from SettingsViewController .. could consolidate 
+        guard let url = URL(string: "https://github.com/settings/connections/applications/\(Secrets.GitHub.clientId)")
+            else { fatalError("Should always create GitHub issue URL") }
+        // iOS 11 login uses SFAuthenticationSession which shares credentials with Safari.app
+        UIApplication.shared.open(url, options: [:])
```

Comment:
Nit: you can omit the `options` parameter as it defaults to an empty Dictionary.